idaes/core/util/tests/test_pid_initialization.py

```diff
@@ -92,27 +96,28 @@ def make_model(horizon=6, ntfe=60, ntcp=2, inlet_E=11.91, inlet_S=12.92):
     )
     # Allegedly the proper energy balance is being used...
 
+    m.fs.pid = PIDController(
+        process_var=m.fs.cstr.volume,
+        manipulated_var=m.fs.cstr.outlet.flow_vol,
+        mv_bound_type=ControllerMVBoundType.SMOOTH_BOUND,
+        calculate_initial_integral=True,
+        # ^ Why would initial integral be calculated
+        # to be nonzero?
+        type=ControllerType.PID,
+    )
+
     # Time discretization
     disc = TransformationFactory("dae.collocation")
     disc.apply_to(m, wrt=m.fs.time, nfe=ntfe, ncp=ntcp, scheme="LAGRANGE-RADAU")
 
-    m.fs.pid = PIDBlock(
-        default={
-            "pv": m.fs.cstr.volume,
-            "output": m.fs.cstr.outlet.flow_vol,
-            "upper": 5.0,
-            "lower": 0.5,
-            "calculate_initial_integral": True,
-            # ^ Why would initial integral be calculated
-            # to be nonzero?
-            "pid_form": PIDForm.velocity,
-        }
-    )
-
-    m.fs.pid.gain.fix(-1.0)
-    m.fs.pid.time_i.fix(0.1)
-    m.fs.pid.time_d.fix(0.0)
+    m.fs.pid.gain_p.fix(-1.0)
+    m.fs.pid.gain_i.fix(10)
+    m.fs.pid.gain_d.fix(0.0)
     m.fs.pid.setpoint.fix(1.0)
+    m.fs.pid.mv_lb.set_value(0.5)
+    m.fs.pid.mv_ub.set_value(5.0)
+    m.fs.pid.mv_ref.fix(0.0)
+    m.fs.pid.derivative_of_error[m.fs.time.first()].fix(0)
```

Comment:
Can you set the type to PI?  Then you shouldn't need this and the derivative gain.